src/core/tsi/ssl_transport_security.c

```diff
@@ -49,6 +49,8 @@
 #include <openssl/x509.h>
 #include <openssl/x509v3.h>
 
+#include "src/core/support/env.h"
```

Comment:
We *could* do it and you're right, the documentation is wrong. However, I can see a case where a server supports both SSL mutual auth and server-only auth and this is why I think we should go for a separate flag in the config.